erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -866,20 +866,22 @@ posix_errno_t efile_read_info(const efile_path_t *path, int follow_links, efile_
     } else {
         HANDLE handle;
         DWORD last_error;
+        DWORD flags;
+
+        flags = FILE_FLAG_BACKUP_SEMANTICS;
+        if(attributes & FILE_ATTRIBUTE_REPARSE_POINT) {
+            flags |= FILE_FLAG_OPEN_REPARSE_POINT;
+            is_link = is_name_surrogate(path);
```

Comment:
I think we only want to open the reparse point if `is_link && !follow_links`, we don't want to open OneDrive markers for example.

Reply:
Ok, I've made this change, and the tests still pass for me. I thought reparse points were just links but I guess it's more involved than that.

Reply:
Thanks, I've updated our tests.